README.md

```diff
@@ -1 +1,148 @@
-# javascript-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Todo List Project](#-Javascript-project)
+  - [🛠 Built With ](#-built-with-)
+  - [🚀 Live Demo video ](https://youtu.be/sgqM34DgA4Q)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Javascript Capstone<a name="about-project"></a>
+
+**javascript Capstone** 
+The JavaScript capstone project  is about building your own web application based on an external API. We select an API (The pokemon api) that provides data about our selected api and we build  the webapp around it. The webapp have 3 user interfaces.
+
+
+## 🛠 Built With <a name="built-with"></a>
+    HTML
+    CSS
+    Javascript(ES6)
+    Webpack
+    AAA patern
+    jest Tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ - [Live Demo video](https://youtu.be/sgqM34DgA4Q)
+
```

Comment:
- [ ] good job for creating a video that describes the project but you should also deploy the project using Heroku or Netlify because not all the hr are tech experts some of them don't know anything about programming so you should have something to show them.